source/OptimaJet.DWKit.StarterApplication/Services/ProjectService.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using JsonApiDotNetCore.Data;
+using JsonApiDotNetCore.Internal;
+using JsonApiDotNetCore.Services;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Optimajet.DWKit.StarterApplication.Models;
+using OptimaJet.DWKit.StarterApplication.Repositories;
+
+namespace OptimaJet.DWKit.StarterApplication.Services
+{
+    public class ProjectService : EntityResourceService<Project>
+    {
+        public IOrganizationContext OrganizationContext { get; private set; }
+        public IJsonApiContext JsonApiContext { get; }
+        public ICurrentUserContext CurrentUserContext { get; }
+        public UserRepository UserRepository { get; }
+        Forms.Projects.CreateForm CreateForm;
+        Forms.Projects.UpdateForm UpdateForm;
+
+
+        public ProjectService(
+            IJsonApiContext jsonApiContext,
+            IOrganizationContext organizationContext,
+            ICurrentUserContext currentUserContext,
+            UserRepository userRepository,
+            IEntityRepository<Project> projectRepository,
+            Forms.Projects.CreateForm createForm,
+            Forms.Projects.UpdateForm updateForm,
+            ILoggerFactory loggerFactory) : base(jsonApiContext, projectRepository, loggerFactory)
+        {
+            OrganizationContext = organizationContext;
+            JsonApiContext = jsonApiContext;
+            CurrentUserContext = currentUserContext;
+            UserRepository = userRepository;
+            CreateForm = createForm;
+            UpdateForm = updateForm;
+        }
+        public override async Task<IEnumerable<Project>> GetAsync()
```

Comment:
there seems to be quite a bit of duplication here with the groups service.

This could probably be abstracted somewhere :)